ztp/siteconfig-generator/siteConfig/siteConfig.go

```diff
@@ -90,31 +95,29 @@ type Clusters struct {
 	ApiVIP                 string            `yaml:"apiVIP"`
 	IngressVIP             string            `yaml:"ingressVIP"`
 	ClusterName            string            `yaml:"clusterName"`
+	ClusterType            string            `yaml:"clusterType"`
 	AdditionalNTPSources   []string          `yaml:"additionalNTPSources"`
 	Nodes                  []Nodes           `yaml:"nodes"`
 	MachineNetwork         []MachineNetwork  `yaml:"machineNetwork"`
 	ServiceNetwork         []string          `yaml:"serviceNetwork"`
-	ManifestsConfig        ManifestsConfig   `yaml:"manifestsConfig"`
-	ClusterType            string            `yaml:"clusterType"`
 	NumMasters             uint8             `yaml:"numMasters"`
 	NumWorkers             uint8             `yaml:"numWorkers"`
-	ClusterProfile         string            `yaml:"clusterProfile"`
```

Comment:
If you're removing it here, should you remove it from the CRD as well?

Reply:
sure